tutorials/tutorials.batch

```diff
@@ -13,98 +13,74 @@
 ~/bin/highlight-pointer -r 10 --key-quit q &
```

Comment:
`highlight-pointer` should be properly installed as a dependency.

Reply:
addressed
https://github.com/Open-MSS/MSS/issues/2375